pkg/client/clientset_generated/internalclientset/typed/batch/unversioned/batch_client.go

```diff
@@ -82,17 +81,9 @@ func setConfigDefaults(config *restclient.Config) error {
 	}
 	// TODO: Unconditionally set the config.Version, until we fix the config.
 	//if config.Version == "" {
-	// XXX why is above commented out?
 	copyGroupVersion := g.GroupVersion
 	config.GroupVersion = &copyGroupVersion
 	//}
-	// Do we need something like this:
```

Comment:
fyi this code is auto-generated so we can't update it manually (that's why these lines are removed)